test/ex_machina/ecto_test.exs

```diff
@@ -100,9 +142,10 @@ defmodule ExMachina.EctoTest do
 
     account = ExMachina.Ecto.assoc(MyApp.EctoFactories, attrs, :account, factory: :user)
 
-    newly_created_account = %{id: 3, admin: false, name: "John Doe"}
+    vals = %{admin: false, name: "John Doe"}
```

Comment:
We are typically calling these `attrs`, so I think it would be more clear if it were changed to `attrs` instead of vals